source/iostreams.tex

```diff
@@ -6838,8 +6838,8 @@
 string out = vformat(os.getloc(), fmt, args);
 \end{codeblock}
 If the function is \tcode{vprint_unicode} and
-\tcode{os} is a stream that refers to a terminal capable of displaying Unicode
-which is determined in an implementation-defined manner,
+\tcode{os} is a stream that refers to \impldef{a terminal capable of displaying
+Unicode} which is determined in an implementation-defined manner,
```

Comment:
Specifically, this appearance of the word "implementation-defined" must be replaced by `\impldef{text to appear in the index}`